packages/core/src/generators/schema.ts

```diff
@@ -41,7 +44,27 @@ export async function generateSchema(
   config.source.initialize?.(config);
 
   // Invoke the content source resolver to retrieve the content nodes
-  const allContentNodes = await config.source.fetch(config.content);
+  let allContentNodes: Record<string, any> = {};
+
+  const addRecord =
+    (source: Source<any>) =>
```

Comment:
[nit] we should be able to use a generic in place of `any` here

Reply:
Yeah, I agree, I originally had it as a generic but I wasn't sure how to set it from the plugin here the way I did with collection